ci_cd/utils/versions.py

```diff
@@ -73,7 +74,7 @@ class SemanticVersion(str):
 
     """
 
-    _REGEX = (
+    _regex = (
```

Comment:
Example (Line 73-83): The regex pattern _REGEX is used for version parsing. Given the introduction of Version, consider whether relying on custom regex is still the optimal approach. The Version class might render this redundant or less reliable.

Reply:
This is a regex for semantic versioning, not python versioning. I will rename the variable to denote this.